### ISSUE TITLE
Plugin: Overlap Buster on axes bullets

### ISSUE THREAD

**Issue by 12rita** [amcharts/amcharts4#4168]
I saw an overlap buster plugin and i think it would fit perfectly to my chart. The question is: if i work with axes bullets what exactly should i push into `overlap.targets` list? I've tried this way:
```
const event = dateAxis.axisRanges.create();
event.date = item.date;
event.bullet = new am4charts.CircleBullet();
overlap.targets.push(event.bullet);
```
But it didn't worked out

**Comment by martynasma** [comment]
Would you be able to post your whole chart and data on CodePen/jsFiddle so that we can test?

**Comment by 12rita** [comment]
Sure: [example](https://codepen.io/12rita/pen/GRGNVmj)

**Comment by martynasma** [comment]
Thanks!

OK, so pushing actual circle element into overlap targets (as opposed the bullet itself) seems to do the trick:

```JavaScript
overlap.targets.push(event.bullet.circle);
```

**Comment by 12rita** [comment]
Indeed, animation has been uppeared, but for some reason when you hover on one bullet every other bullet slides too. Is it supposed to be that? 

Before hover:
![image](https://user-images.githubusercontent.com/18229211/201121534-cf42c85f-7289-46aa-a65c-359de59b0584.png)

After hover: 
![image](https://user-images.githubusercontent.com/18229211/201121685-8e9cf95e-6d13-47e7-b5a0-9feda02f8485.png)

**Comment by martynasma** [comment]
> Indeed, animation has been uppeared, but for some reason when you hover on one bullet every other bullet slides too. Is it supposed to be that?

Well, not intention.

However, OverlapBuster is a generic plugin. Axis bullets are very specific beast, so it must be throwing off positioning.

The buster might not be suitable in this specific case :(

**Comment by 12rita** [comment]
I see, thank you for explanation
